indian_festivals/festival.py

```diff
@@ -68,14 +67,18 @@ def get_festivals_in_a_year(self, month=None):
             festival_details_per_month = festival.findChildren(
                 ['tbody'])[0].findChildren("tr")
             festival_dict[month_name] = []
-
             # Store day, date and name for each festival
             for festival_detail in festival_details_per_month:
                 fests = festival_detail.findChildren("td")
-                festival_dict[month_name].append(
-                    {"date": fests[0].text.strip().split(" ")[0],
-                     "day": fests[0].text.strip().split(" ")[1],
-                     "name": fests[1].text.strip()})
+
+                try:
+                    festival_dict[month_name].append(
+                        {"date": fests[0].text.strip().split(" ")[0],
+                         "day": fests[0].text.strip().split(" ")[1],
+                         "name": fests[1].text.strip()})
+                except Exception as e:
```

Comment:
Reason for putting an exception here ?

Reply:
It giving bug as some time Sr. No also come along with holiday data and when we are spliting it we got an error hence I did exception